homework3/semi-uber/Server/api/utilits/checkforEdit.js

```diff
@@ -0,0 +1,33 @@
+const { Truck } = require("../Schemas/truck.schema");
+const { User } = require("../Schemas/user.schema");
+const { Load } = require("../Schemas/load.schema");
+const { STATUS } = require("../../data/loadData.json");
+
+const checkForEdit = async userID => {
+  const user = await User.findById(userID);
+
+  if (user.status === "Driver") {
+    try {
+      const result = await Truck.findOne({ created_by: userID, edit: false });
```

Comment:
you can try to use ternary operator here for return

Reply:
done